### ISSUE TITLE
 Bluetooth is crashing

### ISSUE THREAD

**Issue by NathanBnm** [elementary/switchboard-plug-bluetooth#77]
See issue here https://github.com/elementary/wingpanel-indicator-bluetooth/issues/84#issue-453773175

**Comment by peteruithoven** [comment]
I don't understand why this would require 2 issues. Does Switchboard also crash when you connect to this device?

**Comment by NathanBnm** [comment]
Yes, since the last update this is what's happening. I didn't know if I had to post it here or on the indicator so I refered both.

**Comment by danirabbit** [comment]
Closing since OP said it was resolved in the other issue
